packages/devtools-launchpad/src/server/launch.js

```diff
@@ -32,8 +40,12 @@ function handleLaunchRequest(req, res) {
   }
 
   if (browser == "Chrome") {
-    ps.spawn(path.resolve(__dirname, "../../bin/chrome-driver.js"),
-     ["--location", location]);
+    const chromeDriver= path.resolve(__dirname, "../../bin/chrome-driver.js");
+    if (isWindows) {
+      ps.spawn('node', [chromeDriver, "--location", location]);
```

Comment:
Wouldn't this also work on other systems? I'm thinking this entire if condition can be removed and instead we could simply always spawn node and provide the arguments in.

Reply:
I am not sure of that, but its agreeable when it produce more cleaner code. Let me try it when i got the time and chance.